### PR TITLE
chore: enables skipped tests, removes some

### PR DESCRIPTION
**Pull request by genehynson**
Went through the skipped cypress tests and found some that could be re-enabled. I ran each test that I enabled 20 times with k8s-idpe and saw no failures.

I also removed a couple tests that no longer seem applicable to the product.

### CODE REVIEW

File: cypress/e2e/cloud/communityTemplates.test.ts [hunk 1]
```diff
@@ -246,12 +246,15 @@ describe('Community Templates', () => {
       cy.go('back')
     })
 
-    // this behavior should be cleaned up
-    it.skip('takes you to github readme when you click on the Community Templates button', () => {
+    it('takes you to github readme when you click on the Community Templates button', () => {
```
Comment (hoorayimhelping) [inline cypress/e2e/cloud/communityTemplates.test.ts:249]: fyi @zoesteinkamp - this is a better solution than what we were doing (i.e. skipping the test)

**Review by hoorayimhelping — APPROVED** [review]
awesome refactoring!

File: cypress/e2e/shared/explorer.test.ts [hunk 2]
```diff
@@ -1167,31 +1169,6 @@ describe('DataExplorer', () => {
       cy.get('.autorefresh-dropdown--pause').click()
       makeGraphSnapshot().shouldBeSameAs(snapshot, false)
     })
-
-    // skip until the auto-refresh feature is added back
-    it.skip('auto refresh', () => {
```
Comment (Asalem1) [inline cypress/e2e/shared/explorer.test.ts:1172]: Not too sure about removing this. Autorefresh is enabled in `oss`, maybe we should turn it on for that environment by moving this test there with a TODO to move it here?

**Review by Asalem1 — APPROVED** [review]

File: cypress/e2e/shared/explorer.test.ts (hunk 2 above)
Comment (genehynson) [reply cypress/e2e/shared/explorer.test.ts:1172]: There were new auto-refresh e2e tests added: https://github.com/influxdata/ui/pull/1304 . So I don't think this skipped test is relevant anymore

Comment (Asalem1) [reply cypress/e2e/shared/explorer.test.ts:1172]: Those are for the dashboard page where the new feature is enabled, but the old feature is enabled in OSS.

Comment (genehynson) [reply cypress/e2e/shared/explorer.test.ts:1172]: Oh I see. Sure, I can move the test to the oss/explorer.test.ts

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 48 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1490)

**Pull request merged**